packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -167,18 +169,16 @@ export class GuxTimeZonePickerBeta {
     );
 
     defaultZoneOptions.forEach(option => {
-      if (
-        defaultZoneOptions.length === 1 &&
-        this.workspaceDefault === this.localDefault
-      ) {
-        defaultZoneOptions[0].defaultZone = `${this.i18n(
-          'localAndWorkspaceDefault'
-        )}`;
+      if (this.workspaceDefault === this.localDefault) {
+        option.defaultZone = `${this.i18n('localAndWorkspaceDefault')}`;
       } else if (option.value === this.workspaceDefault) {
         option.defaultZone = `${this.i18n('workspaceDefault')}`;
       } else if (option.value === this.localDefault) {
         option.defaultZone = `${this.i18n('localDefault')}`;
       }
+      if (option.value === this.customDefault) {
```

Comment:
The zone list is user defined, right? Wondering if the user would ever have an extra space or even a typo in the props: defaultZone and workspaceDefault and then the exact match check here would fail? Maybe that's not our problem and would be on the user?

Reply:
Thats a good question I am not sure as of yet as there was no validation incorporated for the initial properties which were `workspace-default` and `local-default`. I will wait till we are happy with this api change and review this.